packages/react-component-library/src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import Select from 'react-select'
+import { ValueType } from 'react-select/src/types'
+
+import { DropdownIndicator } from './DropdownIndicator'
+import { DropdownLabel } from './DropdownLabel'
+import { DropdownOption } from './DropdownOption'
+
+export interface DropdownProps {
+  onSelect: (value: string) => void
+  options: DropdownOption[]
+  label: string
+}
+
+export const Dropdown: React.FC<DropdownProps> = ({
+  onSelect,
+  options,
+  label,
+}) => {
+  const onChange = (option: ValueType<DropdownOption>) => {
+    const dropdownOption = option as DropdownOption
```

Comment:
The as keyword in TS is new to me. So it is basically just casting the object with that type?